app/views/profile/orders/index.html.erb

```diff
@@ -1,11 +1,24 @@
-<h1>My Orders</h1>
-<% @orders.each do |order| %>
-  <article id="order-<%= order.id %>">
-    <%= link_to "Order ##{order.id}", "/profile/orders/#{order.id}" %>
-    <p>Date created: <%= order.created_at %></p>
-    <p>Date updated: <%= order.updated_at %></p>
-    <p>Status: <%= order.status %></p>
-    <p>Item quantity: <%= order.items.length %></p>
-    <p>Grand total: <%= order.grandtotal %></p>
-  </article>
-<% end %>
\ No newline at end of file
+<h1 align = "center">My Orders</h1>
```

Comment:
This all looks so clean!

Reply:
I pretty much copy/pasted the existing /orders/:id page lolol. All thanks to Meg for the "template".